test/spec.yml

```diff
@@ -201,35 +232,41 @@
 
 - test: Test login (bad-path)
   steps:
-    -   in: bn create stormysummer
+    -   in: bn create node8 stormysummer
     -   in: echo "9239239" | bn login
         err: Invalid API key
         exit: 1
 
 - test: Test create (bad-path)
   steps:
-    -   in: bn create a*b*c*
+    -   in: bn create node8 a*b*c*
         err: Invalid characters in function name a*b*c*. Use only letters and digits
         exit: 1
-    -   in: bn create a_b%c#d@e+
+    -   in: bn create node8 a_b%c#d@e+
         err: Invalid characters in function name a_b%c#d@e+. Use only letters and digits
         exit: 1
-    -   in: bn create a=b~c,d.
+    -   in: bn create node8 a=b~c,d.
         err: Invalid characters in function name a=b~c,d.. Use only letters and digits
         exit: 1
-    -   in: bn create a:b?c!d-
+    -   in: bn create node8 a:b?c!d-
         err: Invalid characters in function name a:b?c!d-. Use only letters and digits
         exit: 1
-    -   in: bn create tooooooooooooooooooooooooooooooooooooooooooooooolongggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
+    -   in: bn create node8 tooooooooooooooooooooooooooooooooooooooooooooooolongggggggggggggggggggggggggggggggggggggggggggggggggggggggggg
         err: Function names cannot be longer than 58 characters.
         exit: 1
     -   in: bn create
-        err: "*Not enough non-option arguments: got 0, need at least 1"
+        err: "*Not enough non-option arguments: got 0, need at least 2"
+        exit: 1
+    -   in: bn create node8
+        err: "*Not enough non-option arguments: got 1, need at least 2"
+        exit: 1
+    -   in: bn create abcd
```

Comment:
can't find a spec for `bn create java8 hello`?
(which will verify the `No template for runtime xxx` error)

Reply:
Added test for cpp4096